backend/pkg/helm/release_test.go

```diff
@@ -74,15 +101,18 @@ func TestInstallRelease(t *testing.T) {
 
 	helmHandler.installRelease(installReq)
 
-	status, err := actionState.GetStatus("install", "helm-test-asdf")
+	status, err := getStatus(cache, "install", "helm-test-asdf", t)
```

Comment:
Please change `getStatus` parameters accordingly for all iterations as suggested change above.